### PR TITLE
Finish Timezone Picker UI

### PR DESCRIPTION
**Pull request by carlodicelico**
This PR finishes up implementing #35 with awesome UI work from @erin-doyle! Sorry for some of the commit messages, where I was wrestling with git's core.ignorecase 😜

### CODE REVIEW

**Comment by ecmadao** [comment]
@carlodicelico Calm down😂
I notice that only minute can not fetch, with 1 mintue as difference value (expect 39, get 40)
Maybe because of travis ci build delay? Cause it only failed for one test add succeed for another one.

**Comment by ecmadao** [comment]
I rebuild test and all passed 😉

**Comment by ecmadao** [comment]
@carlodicelico 
Almost half reviewed!🚀
After review finished and merged, I'll change README and ChangeLog and publish a new version.
Plan to done these things during the daytime.

File: examples/TimeZonePickerWrapper.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```
Comment (ecmadao) [inline examples/TimeZonePickerWrapper.js]: `examples/TimeZonePickerWrapper.js` is duplicate with `examples/TimezonePickerWrapper.js` ?

**Comment by ecmadao** [comment]
I notice you have already add `phrases` in README, cool!
But it seems I can not find any func like `handleTimezoneChange` send to `MaterialTheme` or `ClassicTheme` in `components/TimePicker.jsx`, so `handleTimezoneChange` will always be `() => {}` and user can't get new timezone.

**Comment by carlodicelico** [comment]
Ah, nice catch! We'll fix that today.

File: examples/TimeZonePickerWrapper.js (hunk 1 above)
Comment (carlodicelico) [inline examples/TimeZonePickerWrapper.js]: Renamed it.

**Comment by ecmadao** [comment]
Merge! Thanks you all! @carlodicelico @erin-doyle 👍👍👍👍

**Comment by ecmadao** [comment]
New version v2.1.0 publish!

**Pull request merged**